bin/regen-screenshots

```diff
@@ -260,7 +260,7 @@ class RegenScreenshots
         name: "blog-app#{i + 1}",
         unix_user: "a",
         public_key: "a",
-        location: "hetzner-fsn1",
+        location_id: "caa7a807-36c5-8420-a75c-f906839dad71",
```

Comment:
We should probably be consistent and use the same approach as earlier in this script.